### PR TITLE
update budget from 100 to 30

### PR DESCRIPTION
**Pull request by irvin**

### CODE REVIEW

**Comment by MichaelKohler** [comment]
@couci do you approve?

**Review by MichaelKohler — APPROVED** [review]
This can be merged once Konstantina approves the text change. Thanks for your contribution!

**Comment by couci** [comment]
yes approved, thanks @irvin

**Pull request merged**
